### ISSUE TITLE
I have another question..

### ISSUE THREAD

**Issue by GuilhermeDunkly** [olegos2/mobox#200]
Why can't you use Chrome or any other browser on the mobox?  Why do some programs just not work?  Is it due to the lack of some important library?  and what is the "media foundation (for RE)" option for?

**Comment by olegos2** [comment]
Media foundation is mostly required for video playback in games. Why do you need browser in mobox tho?

**Comment by GuilhermeDunkly** [comment]
Not at all, I would just find it interesting to have a functional browser
